src/calculateRentalCost.js

```diff
@@ -4,7 +4,33 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  // I tried everyting, and there is actualy second way to solve this problem
+  // Well, this program gets job done.
+
+  const LONG_TERM = 7;
+  const MED_TERM = 6;
+  const SHORT_TERM = 3;
+
+  const BASE_PRICE = 80;
+  const SHORT_TERM_DISCOUNT = -20;
+  const MID_TERM_DISCOUNT = -140;
+  const LONG_TERM_DISCOUNT = -150;
+
+  if (days < SHORT_TERM) {
+    return BASE_PRICE;
+  }
+
+  if (days >= SHORT_TERM && days <= MED_TERM) {
+    if (days === SHORT_TERM) {
+      return BASE_PRICE - SHORT_TERM_DISCOUNT;
+    } else if (days === MED_TERM) {
+      return BASE_PRICE - MID_TERM_DISCOUNT;
+    }
+  }
```

Comment:
The conditionals inside the if block are unnecessary and overly specific. You should calculate the total price based on the number of days and apply the SHORT_TERM_DISCOUNT if the days are between SHORT_TERM and MED_TERM inclusive. There's no need for the check against MED_TERM as it's not part of the task requirements.